BEIMA.Client/src/shared/ItemList/ItemList.js

```diff
@@ -48,7 +48,7 @@ const Item = ({item, RenderItem, isDeviceList}) => {
   return (
     <div className={styles.item}>
       <div className={styles.row}> 
-        {isDeviceList ? <div className={styles.itemName}>{item.deviceTag} - {"<Device Type Name>"} - {"<Building Name>"}</div> : <div className={styles.itemName}>{item.name}</div>}
+        {isDeviceList ? <div className={styles.itemName}>{item.deviceTag} - {item.deviceTypeName} - {"<Building Name>"}</div> : <div className={styles.itemName}>{item.name}</div>}
```

Comment:
I think putting the device type in parenthesis might look better. Like "DeviceName (Device Type)" or something, open to other ideas as well. Or maybe we have a different task for formatting on the pages.

Reply:
Yeah we could put formatting into a different task. I know we will need to do some UI clean up, and that would be a good task to do it